src/components/footer/footer.js

```diff
@@ -22,35 +23,34 @@ const socialLinks = [
 
 const Footer = () => {
   return (
-    <footer className="custom-footer">
-      <div className="custom-footer__logo">
-        <Logo />
-      </div>
-      <div className="links-to-pages">
-        {links.map(link => (
-          <Link key={link.id} to={link.path} className="nav-link link">
-            <i className="link-circle fas fa-circle link-text" />
-            {link.name}
-          </Link>
-        ))}
+    <footer className="footer__container">
+      <div className="footer__links_and_logo_container">
+        <Link to="/home">
+          <img src={LogoWhite} alt="Logo" className="footer__logo" />
+        </Link>
+        <div className="footer__links_to_pages_container">
+          {links.map(link => (
+            <Link key={link.id} to={link.path} className="footer__link">
```

Comment:
Can you drop id property from mapper and use path for key?

Reply:
Done